src/bitfield.cpp

```diff
@@ -42,20 +42,24 @@ POSSIBILITY OF SUCH DAMAGE.
 
 namespace libtorrent {
 
-	bool bitfield::all_set() const noexcept
+	bool bitfield::all_set() const noexcept //all set 
 	{
-		if(size() == 0) return false;
+		if(size() == 0) return false; //checks if all bits in the bitfield are set to 1. If the size of the bitfield is zero, it returns false.
```

Comment:
This comment is confusing in this place.

Reply:
yeah, this comment does not make sense. It would be more sensible to just indicate that it's on purpose that a bitfield with 0 length is not considered have any bits set